lib/ansible/module_utils/api.py

```diff
@@ -114,3 +116,51 @@ def retried(*args, **kwargs):
 
         return retried
     return wrapper
+
+
+def generate_jittered_backoff(retries=10, delay_base=3, delay_threshold=60):
```

Comment:
This is great. Much better than the exponential backoff I implemented for `reboot`.

Reply:
Thanks. After this is merged I'm going to refactor the other places that have implemented retries to use common code (apt, apt_repository, reboot, and the GalaxyAPI import process).